### ISSUE TITLE
Did you mean ```static observedProperties = ['title', 'body'];``` here?

### ISSUE THREAD

**Issue by kenchris** [PolymerLabs/lit-html#34]
Se below

```
class MyElement extends CoolLitMixin(HTMLElement) {

  static observedProperties = ['message', 'name'];

  title = `About lit-html`;
  body = `It's got potential.`;

  // Called by the base-class when properties change, result is passed
  // to lit-html's render() function.
  render() {
    return html`
      <h1>${this.title}</h1>
      <p>${this.body}</p>
    `;
  }
}
```

**Comment by justinfagnani** [comment]
Oops, thanks!

**Comment by justinfagnani** [comment]
Fixed in d36e6af4c3cfe9023c9cb6a625a48a782af15bb0
